module/x/gravity/keeper/querier.go

```diff
@@ -13,7 +13,7 @@ import (
 
```

Comment:
This whole file needs to go 🗡️ we are only going to be relying on the GRPC query stuffs.

Reply:
Done